service/history/queues/executable.go

```diff
@@ -456,6 +457,8 @@ func (e *executableImpl) rescheduleTime(
 		// as the error means mutable state is not ready to handle the task,
 		// need to wait for replication.
 		e.timeSource.Now().Add(taskNotReadyReschedulePolicy.ComputeNextDelay(0, attempt))
+	} else if err == consts.ErrDeleteOpenExec {
+		e.timeSource.Now().Add(deleteOpenExecutionReschedulePolicy.ComputeNextDelay(0, attempt))
```

Comment:
Sorry I think the original code is buggy. I should return the value on L459. I will send out a PR to fix that.

Reply:
Oh I see, good catch!